app/server/index.js

```diff
@@ -294,11 +297,31 @@ app.prepare().then(() => {
     return handle(req, res);
   });
 
-  server.listen(port, (err) => {
-    if (err) {
-      throw err;
-    }
+  if (process.env.HOST.startsWith('https')) {
```

Comment:
`process.env.HOST` is `undefined` in our e2e test jobs. 
```suggestion
  if (process.env.HOST && process.env.HOST.startsWith('https')) {
```

Reply:
goddamn prettier. I wrote `/^https/.test(process.env.HOST)`. fixing